terraform/modules/quorum-cluster-region/quorum-cluster-region-backup-event.tf

```diff
@@ -0,0 +1,399 @@
+
+# This is to be placed in quorum-cluster-region
+
+provider "archive" {}
+provider "random" {}
+provider "null" {}
+
+data "local_file" "private_key" {
+  filename = "${var.private_key_path}"
+}
+
+resource "aws_s3_bucket_object" "object" {
```

Comment:
Can this name be more descriptive than `object`?

Reply:
fixed